### DISCUSSION TITLE
Festum SS. Nominis Jesu: how is it determined?

### DISCUSSION THREAD

**Discussion by Augustinus-Altovadensis** [DivinumOfficium/divinum-officium#3611] [Q&A]
Laudétur Jesus Christus!

Currently I'm a bit behind and working on the start of year and I've hit a wall: I cannot get the feast of the Name of Our Lord appear in my Cistercian calendar. It works everywhere apart from the Monastic 1963 Office. However, currently the Cistercian Office isn't called _Cistercian Monastic_ anymore, so it shouldn't be linked to that. I tried exchanging the `data.txt `entries in `Tabulae` folder, of course I tried to look in the code, but in all of this I have come surprisingly empty-handed. 

So my question is: how is the feast of the Name of Our Lord determined, how does the code tell that it's the $winner? 

I know that the current rubric is quite simple: if there is a Sunday between January 2 and 5, it falls there, otherwise on the January 2., but I cannot seem to get it working on the Cistercian version.

Thanks for any help, God bless!

**Comment by APMarcello3** [comment]
Have you tried using file Sancti/01-00.txt?

**Reply by Augustinus-Altovadensis** [reply]
Actually, after changing the `TemporaCist/Nat2-0.txt` to SS. Nominis and it suddenly works, although before, I wasn't able to see the Second Sunday in Nativity-tide... Strange. And of course, I have the `Sancti(Cist)/01-00.txt` file. I was quite interested, how it actually works (I still am), but now I'm glad that it works :-)
